### ISSUE TITLE
Join Club Button

### ISSUE THREAD

**Issue by KaosElegent** [Seneca-Software-Developers-Club/ssdc-website#15]
2 Scenarios to code:
- Every page on mobile
- Home page on desktop

### Mobile
- `Join Club` in the navbar

### Desktop home page:
- `Join Club` + Socials + Footer, all at the bottom

### `Join Club` + Socials:
![Image](https://github.com/user-attachments/assets/781830a5-ae21-4086-af39-bc55dc9b73c8)

The join link: `https://clubs.ssfinc.ca/SSDC/club_signup`
You can make a component for a `Button` that lets you customize `text`, `bg-color`, `font color`, `font size` and use in `/contact`

**Comment by raghavdotdev** [comment]
My question regarding the Join button is: Are we only responsible for its design, or do we also need to implement its functionality? If functionality is required, how should it work?

**Comment by KaosElegent** [comment]
> My question regarding the Join button is: Are we only responsible for its design, or do we also need to implement its functionality? If functionality is required, how should it work?

Yeah, you need to implement the functionality as well. it should link to `https://clubs.ssfinc.ca/SSDC/club_signup`. I'll add this detail to the ticket as well. Thanks! This is the link that should've been used in the `/contact` page as well btw.

**Comment by KaosElegent** [comment]
If this is in-progress, you should move it to that column/change the status 🙃
